docs/en/sql-reference/sql-statements/Data Definition/SHOW RESOURCES.md

```diff
@@ -25,17 +25,19 @@ under the License.
 -->
 
 # SHOW RESOURCES
```

Comment:
I think we should use system table instead of show command....
For example, select * from information_schema.resources where xxxx;

Reply:
This is a good suggestion, this pr is mainly for adding s3 resource, using the historical resource design. 
We can do a code refactoring for show resource after this is done. 
This would make the code more independent and would make the review easier